### ISSUE TITLE
Option Menu Freezes

### ISSUE THREAD

**Issue by ellie12134** [nipkownix/re4_tweaks#220]
### Validation

- [X] Game has been updated to the latest RE4HD release (v1.0.1: https://www.re4hd.com/?p=9471)
- [X] Data has been [validated with SFV](https://www.re4hd.com/?p=9454) (a guide to using QuickSFV is available at that link, use that with this 1.0.1 SFV: [BIO4-HDProject1.0.1-SFV.zip](https://github.com/nipkownix/re4_tweaks/files/8057899/BIO4-HDProject1.0.1-SFV.zip))

### Describe your issue here (drag+drop ZIP to attach it)

After installing the latest RE4_Tweaks, the options menu tends to freeze a lot when using it during gameplay, and there is no way to unfreeze it , forcing me to close the game via task manager, which is a pain in the ass because it's extremely difficult to alt tab the game in that state

(I don't use RE4HD, I dont like HD mods, still I had to check the RE4HD mark in the subtmit issue page)

**Comment by emoose** [comment]
Do you mean the in-game options or the re4_tweaks options?

If it's the in-game options, could you try removing the `Bin32\re4_tweaks\sideload` folder and see if that makes any difference? Could be something up with the way we're enabling the subtitles option when playing in english, removing that folder should stop it from trying to enable it.

**Comment by ellie12134** [comment]
yeah it's the in-game options, I will attempt and try to do what you said to see if it worked , after thatt I will upload the log file here

**Comment by ellie12134** [comment]
[dinput8.log](https://github.com/nipkownix/re4_tweaks/files/8923572/dinput8.log)

**Comment by ellie12134** [comment]
Here's the log, I haven't tested your method yet because I had major issues with the pc, gonna test now

**Comment by ellie12134** [comment]
I tried deleting the subtitles, testing , and then reinstalling them and testing, both times worked, if it happens again I'll let you know

**Comment by Mister-Curious** [comment]
Are you using the Debug menu at all? Sometimes if we enter this menu and don't escape it, and bring up other menus there is a freeze of sorts. I just remember to exit the Debug before doing anything else.
